src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/SpaceManagementTest.java

```diff
@@ -134,26 +132,14 @@ void GIVEN_user_component_config_with_space_management_WHEN_space_exceeds_THEN_e
             }
         };
         try (AutoCloseable l = createCloseableLogListener(listener)) {
-            for (int i = 0; i < 14; i++) {
+            for (int i = 0; i < 15; i++) {
                 createTempFileAndWriteData(tempDirectoryPath, "integTestRandomLogFiles.log_", "");
             }
             assertTrue(cdl.await(60, TimeUnit.SECONDS), "5 files deleted");
```

Comment:
Ugh.. this test is not great... it is checking for log messages to see if a file got deleted. Could we add a TODO on line 127 

```suggestion
          // TODO: We should not be checking for log messages to verify if a log file was deleted.
```

Also it is not verifying the active file is still present